src/network/network_client.cpp

```diff
@@ -868,6 +868,8 @@ NetworkRecvStatus ClientNetworkGameSocketHandler::Receive_SERVER_MAP_DONE(Packet
 	/* Say we received the map and loaded it correctly! */
 	SendMapOk();
 
+	ShowClientList();
```

Comment:
Shuold this be here? Or in the SPECTATOR case below only?
Thinking of cases
* Starting a non-dedicated server yourself.
* Joining a company directly via command line or other options.

Reply:
As talked over on IRC:

We could either open it only when you join as spectator, or always. We currently do always, but it is a bit of a toss-up which is better.